health_vara.py

```diff
@@ -64,9 +70,24 @@ def search_mobile(cls, _name, clause):
         value = clause[2]
 
         res.append(('contact_mechanisms.type', '=', 'mobile'))
-        res.append(('contact_mechanisms.value', op, value))
+        res.append(('contact_mechanisms.rec_name', op, value))
         return res
 
+    @classmethod
+    def search_rec_name(cls, name, clause):
```

Comment:
Is this syntax that we have invented or a general syntax for search in Tryton? What is the format of the search query in `clause`?